src/js/components/form/FieldSelect.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const FieldSelect = (props) => {
+  return (
+    <span className="form-control form-control-select">
```

Comment:
Couldn't this be directly on the select? That's what we do for other `form-control` elements